docs/admin-guide/install-buildout.md

```diff
@@ -31,14 +31,12 @@ For other installation options, see {ref}`get-started-install-label`.
 
 ## Prerequisites for installation
 
--   For Plone 6.0, Python {{SUPPORTED_PYTHON_VERSIONS_PLONE60}}
-% TODO: These instructions install Plone 6.0.x. Uncomment next line and change the subsequent include when Plone 6.1 is released and "latest".
-% -   For Plone 6.1, Python {{SUPPORTED_PYTHON_VERSIONS_PLONE61}}
+-   For Plone 6.1, Python {{SUPPORTED_PYTHON_VERSIONS_PLONE61}}
```

Comment:
@mauritsvanrees Do we need to change 6-latest to 6.1-latest on line 63, or will 6-latest be updated to point to 6.1?

Reply:
I will update 6-latest to 6.1-latest.